core/arch/arm/mm/core_mmu.c

```diff
@@ -158,8 +158,9 @@ static void core_mmu_mmap_init(struct tee_mmap_region *mm, size_t max_elem,
 
 	for (n = 0; map[n].type != MEM_AREA_NOTYPE; n++) {
 		mme.pa = map[n].pa;
-		mme.va = map[n].pa;
+		mme.va = map[n].va;
```

Comment:
Sorry. I can not follow you. From my understanding, if using 1 to 1 mapping, nothing needs to be changed. But if we need to support non 1 to 1 mapping, then need to use va not pa.